libnavui-dropin/src/test/java/com/mapbox/navigation/dropin/testutil/TestLifecycleOwner.kt

```diff
@@ -6,7 +6,7 @@ import androidx.lifecycle.LifecycleRegistry
 
 internal class TestLifecycleOwner : LifecycleOwner {
     @Suppress("MemberVisibilityCanBePrivate")
-    val lifecycleRegistry = LifecycleRegistry(this)
+    val lifecycleRegistry = LifecycleRegistry.createUnsafe(this)
```

Comment:
Nice!

Reply:
There is a standard [`TestLifecycleOwner`](https://developer.android.com/reference/kotlin/androidx/lifecycle/testing/TestLifecycleOwner), where I learned this trick, and which can probably be used instead of own implementations (there are 7 of them, by the way).